### DISCUSSION TITLE
I am not able to open the codespace vscode  " open in browser  "

### DISCUSSION THREAD

**Discussion by AshishYesale7** [community/community#72786] [Codespaces]
### Select Topic Area

Bug

### Body

Is there any way to troubleshoot the codespace probem   which shows "Setting up your codespace"  and black screen appears .
<img width="1440" alt="Screenshot 2023-10-28 at 6 51 41 PM" src="https://github.com/community/community/assets/67554129/a9a840e4-dc5b-441e-b6cf-57995c7350f8">

**Comment by ktauchathuranga** [comment]
try using different browser 🖤
